### DISCUSSION TITLE
Can not create server because ping attempts failed on master server (port 28802)

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1218] [Help and Support]
I created a server,first I got the message: Master server reply: server updated (port 28801), sending ping request (on port 28802).
But then I got:  Master server error: ping attempts failed (tried 5 times on port 28802). Server will not be listed.
I have a router,all ports are opened by using DMZ. What's wrong? DO I NEED DDNS?

**Comment by qreeves** [comment]
You do not need a dynamic DNS service. There are many reasons this could be happening.

You might need to check your operating system firewall as well.

Also note using DMZ without understanding the risks is not recommended, you should only open the ports you need and let the router handle firewalling.

**Comment by Calinou** [comment]
Make sure you've forwarded ports in UDP, not TCP. (Forwarding both TCP and UDP works too, but is unnecessary.)

I'm not sure if the range of ports to forward is 28800-28802 or 28801-28802, still.

**Comment by ghost** [comment]
"Master server reply: server updated (port 28801)" means I connected successfully or not? Because when I close the port fowarding this message still appear. (firewall is added these ports)

**Comment by Jigoku** [comment]
To confirm if it's working, you'll want to look for this message:

```
2020-11-13 02:31.25 Master server reply: server registered (port 28801), sending ping request (on port 28802)
2020-11-13 02:31.25 Master server reply: ping reply confirmed (on port 28802), server is now listed
```

If you don't see that message, it's most likely a firewall issue as mentioned above.

**Comment by qreeves** [comment]
While you may be able to connect to the master server, what it is doing is making sure other people can connect to you, which is a very different scenario. If other people can't connect to your server, it would be misleading to list it on the server list.

We are unable to help beyond the advice we have already given here, as every router is different. Your best bet is to Google the name and model number of your router with the words "port forward". The issue is with your network configuration, not Red Eclipse.

EDIT: Also, if you're using Windows, be sure Windows Firewall is configured to allow access to `redeclipse_server.exe` on the same ports.

**Comment by ghost** [comment]
I finally connected, but why I can't find my server in game list, I can find it  only in Redflare, my server only appeared when Search LAN for servers is enabled? And how to rename the server?

**Comment by Jigoku** [comment]
> I finally connected, but why I can't find my server in game list, I can find it only in Redflare, my server only appeared when Search LAN for servers is enabled? And how to rename the server?

Try forcing an update of the list with the button at the top of the UI. It may take a few seconds for some servers to appear at times.
To rename the server you'd want to adjust the settings for these commands:
```
sv_serverdesc "My server"
sv_servermotd "Welcome to my server!"
```

There's also an example servinit.cfg file here; https://github.com/redeclipse/base/blob/master/doc/examples/servinit.cfg with some comments about some basic settings.
